BUILD.bazel

```diff
@@ -8,8 +8,9 @@ cc_library(
         "include/**/*.hpp",
         "include/**/*.h",
         "include/**/*.ipp",
-    ]),
+    ]) ,
```

Comment:
hey I've been noticing a few formatting issues in some of the build files you're sending occasionally. Are you using `buildifier`? If not then I'd highly recommend setting it up in your editor!